### DISCUSSION TITLE
Dockerfile.cuda build fail

### DISCUSSION THREAD

**Discussion by y30n9ju1v** [TabbyML/tabby#1741] [Q&A]
Hi
I am trying to build docker image on my computer
but I got build fail message 
error: Could not find `Cargo.toml` in `/root/workspace`or any parent directory
Is there anything else that needs to be done?

**Comment by wwayne** [comment]
Hi @y30n9ju1v  could you please share the build command you are using?

Previously I built an image on Windows computer with the following Dockerfile, it is used for building an image that contains a pre-downloaded model inside.
```
FROM tabbyml/tabby
ENV TABBY_MODEL_CACHE_ROOT=/models
RUN /opt/tabby/bin/tabby-cpu download --model StarCoder-1B
```

Then I did `docker build -t tabby-offline .`

**Reply by y30n9ju1v** [reply]
@wwayne I used this Dockerfile https://github.com/TabbyML/tabby/blob/main/docker/Dockerfile.cuda
and I just typed `docker build -t tabbyml:0.1 -f <docker file> .`
thank you for your quick reply.

**Comment by y30n9ju1v** [comment]
with whole tabby repository codes and --no-cache option, I got docker image.
